spec/results-model-spec.js

```diff
@@ -15,7 +15,10 @@ describe("ResultsModel", () => {
     atom.project.setPaths([path.join(__dirname, "fixtures/project")]);
 
     editor = await atom.workspace.open("sample.js");
-    resultsModel = new ResultsModel(new FindOptions());
+    reporterSpy = {
+      sendSearchEvent: sinon.spy()
```

Comment:
I've seen most packages and Atom core using `sinon` for mocking/spying instead of the builtin jasmine methods (maybe because the Jasmine version shipped in Atom is ancient?), but I can easily change that 👍